src/common/search/FilterPlays.jsx

```diff
@@ -80,19 +105,26 @@ const FilterPlays = () => {
     labels: [],
     creators: [],
   });
+  const [noOfAppliedFilter, setnoOfAppliedFilter] = useState(0);
+
   useBackListener(({ action }) => {
     if (action === "POP") {
       console.log("POP");
       setModifiedFilterQuery({
         level: "",
         tags: [],
         creator: "",
+        labels: [],
+        creators: [],
       });
       setFilterQuery({
         level: "",
         tags: [],
         creator: "",
+        labels: [],
+        creators: [],
```

Comment:
Same question for creators

Reply:
It's been taken as an array in order to support for multi labels and creators' filter